py/torch_tensorrt/_Input.py

```diff
@@ -384,4 +384,21 @@ def example_tensor(
                 raise ValueError(
                     "Requested an example tensor from a dynamic shaped input but did not specific which profile field to use."
                 )
+        type = self.dtype.to(torch.dtype, use_default=True)
+        if zero_filled:
+            return torch.zeros(shape, dtype=type)
+        else:
+            if self.dtype in [dtype.u8, dtype.i8, dtype.i32, dtype.i64]:
+                return torch.randint(
+                    max(torch.iinfo(torch.int8).min, torch.iinfo(type).min),
```

Comment:
unary like ACOS does not support int32 in the TRT, but supports int8. Using 8bit range random integer to cover all int type.